commands/netlify.go

```diff
@@ -34,10 +39,15 @@ func Execute() {
 	rootCmd.PersistentFlags().BoolVarP(&operations.AssumeYes, "yes", "y", false, "automatic yes to confirmation prompts")
 
 	addCommands()
-
 	if c, err := rootCmd.ExecuteC(); err != nil {
 		if isUserError(err) {
 			c.Println(c.UsageString())
+		} else if uErr, ok := err.(*aoperations.ListSitesDefault); ok {
```

Comment:
when does this happen? I wonder if we can make this a separate function or put it somewhere else because I'm guessing we'll want more error handling logic like this.